waterlog/src/Components/TankGraph.js

```diff
@@ -47,36 +48,37 @@ const TankGraph = (props) => {
       xAxes: [{
         scaleLabel: {
           display: true,
-          labelString: 'Days',
+          labelString: 'days',
           fontColor:'rgb(236,239,241)'
 
         },
         ticks: {
           fontColor:'rgb(236,239,241)',
           major: {
             fontStyle: 'bold',
-            fontColor: 'rgba(0,2,5,0.4)'
+            fontColor: '#000205'
           }
         },
         gridLines: {
           display: false,
-          color:'rgb(236,239,241)'
+          color:'#ECEFF1'
         }
       }],
       yAxes: [{
         display: true,
         scaleLabel: {
           display: true,
-          labelString: 'PercentageLevel',
-          fontColor:'rgb(236,239,241)'
+          labelString: 'percentage level',
+          fontColor:'#ECEFF1'
         },
         ticks: {
           fontColor:'rgb(236,239,241)',
+        },
         gridLines: {
           display: false,
           color:'rgb(236,239,241)'
```

Comment:
above changes was made from rgb to hex. why is this still rgb? Also, why was is changed to hex?

Reply:
The reason is that they said we have to use one either rgb ox Hex ,,,,The majority in our file were in hex so I've changed that to  hex as well.

Reply:
the reason is that the majority that is used is hex. and they said i must changed it to hex .So I've changed that rgb to hex..